docs/source/savant_101/12_video_processing.rst

```diff
@@ -492,3 +493,23 @@ To configure conditional encoding, add a subsection to ``output_frame`` as follo
         tag: <tagname, e.g. encode>
 
 An example of conditional drawing can be found in a dedicated Savant `sample <https://github.com/insight-platform/Savant/tree/develop/samples/conditional_video_processing>`__.
+
+Pass-through mode
+-----------------
+
+Pass-through mode is a special mode when the module doesn't encode the frame but passes it to the sink as is. This mode is useful when your module doesn't modify the frame but only adds some metadata to it.
+
+To configure pass-through mode, set ``output_frame.codec`` to ``copy``:
+
+.. code-block:: yaml
+
+    output_frame:
+      codec: copy
+
+.. note::
+
+    Conditional encoding is ignored in pass-through mode.
```

Comment:
@tomskikh please revert that. I was mistaken. Let us keep conditional video encoding for pass-through mode too.

Reply:
Reverted